### DISCUSSION TITLE
Slack Channel

### DISCUSSION THREAD

**Discussion by abkfenris** [xarray-contrib/xpublish#171] [General]
We've started an Xpublish Slack channel on ESIP's instance for general discussion:
1. ~[ESIP Slack invite](https://esip-all.slack.com/join/shared_invite/zt-1omjufm9z-iH8Gf7gmmsm2SiS5Xh6BlQ#/shared-invite/email)~ Ask here to be invited, the ESIP admins have locked things down more.
2. Then find your way to `#xpublish`
3. ~There is no step 3~

**Comment by kthyng** [comment]
@abkfenris I would like to join so I can discuss more specific questions. But, I don't have an email address that will allow me to join automatically. Can you invite me?

**Reply by abkfenris** [reply]
I'm out on an adventure right now, but I added myself a reminder to invite you when I'm back next week. If you don't have an invite by the end of the week give me another ping.

**Reply by abkfenris** [reply]
I've sent an invite, but I guess the admins need to approve it.

**Reply by kthyng** [reply]
I'm in, thanks!

**Reply by abkfenris** [reply]
Great. Now you can enjoy how awkwardly quiet it seems to be in there recently.

**Comment by jonmjoyce** [comment]
Is there a way to get on the ESIP slack with an @tetratech.com email address?
